MovieDB/Scenes/Movie/MoviesViewModel.swift

```diff
@@ -95,6 +99,24 @@ extension MoviesViewModel: ViewModelType {
             })
             .mapToVoid()
         
+        let selectedBanner = input.selectedBannerTrigger
+            .withLatestFrom(movieBannerList) {
+                return ($0, $1)
```

Comment:
```suggestion
                return $1[$0.row]
```
remove 
` .map { (indexPath, movieBanners) in
                return movieBanners[indexPath.row]
            }
`

Reply:
Modified & removed🙇‍♂️